### PR TITLE
US140512: upgrade some components to lit2 version

### PR DESCRIPTION
**Pull request by alexli01**
This is part of the work to upgrade QED to use lit2 components. It was having some conflict with the lit1 version of components used by d2l-question.

### CODE REVIEW

File: quiz/d2l-quiz-question-hint.js [hunk 1]
```diff
@@ -1,7 +1,5 @@
 import { PolymerElement } from '@polymer/polymer/polymer-element.js';
 
-import 'd2l-icons/d2l-icon.js';
```
Comment (ianmclean2011) [inline quiz/d2l-quiz-question-hint.js:3]: Were these just unnecessary here?

File: demo/index.html [hunk 2]
```diff
@@ -36,7 +38,7 @@
 		const $_documentContainer = document.createElement('template');
 
 		$_documentContainer.innerHTML = `<custom-style include="d2l-typography">
-			<style is="custom-style" include="d2l-typography"></style>
+			<style is="custom-style"></style>
```
Comment (ianmclean2011) [inline demo/index.html:41]: Can you just get rid of this internal style element?

File: package.json [hunk 3]
```diff
@@ -35,14 +40,11 @@
   },
   "main": "d2l-question.js",
   "dependencies": {
-    "@brightspace-ui/core": "^1.85.1",
+    "@brightspace-ui/core": "^2",
     "@polymer/polymer": "^3.0.0",
-    "d2l-button": "BrightspaceUI/button#semver:^5",
```
Comment (ianmclean2011) [inline package.json:40]: Are there any additional button styles or anything that we're grabbing from here?

File: demo/index.html (hunk 2 above)
Comment (ianmclean2011) [reply demo/index.html:41]: Also, when do we use this file? Maybe just kill this?

Comment (alexli01) [reply demo/index.html:41]: This is a demo file. We can just leave it as it is.

File: package.json (hunk 3 above)
Comment (alexli01) [reply package.json:40]: No.

File: quiz/d2l-quiz-question-hint.js (hunk 1 above)
Comment (alexli01) [reply quiz/d2l-quiz-question-hint.js:3]: Good catch, I am supposed to replace it with a brighspace icon import. I will fix it

File: demo/index.html (hunk 2 above)
Comment (ianmclean2011) [reply demo/index.html:41]: Sorry ... I thought this was QED ... didn't realize it was a component file.

**Review by ianmclean2011 — APPROVED** [review]

**Pull request merged**